chia/rpc/wallet_rpc_client.py

```diff
@@ -246,6 +247,10 @@ async def create_signed_transaction(
             coins_json = [c.to_json_dict() for c in coins]
             request["coins"] = coins_json
 
+        if exclude_coins is not None and len(exclude_coins) > 0:
+            exclude_coins_json = [exclude_coin.to_json_dict() for exclude_coin in exclude_coins]
+            request["exclude_coins"] = exclude_coins_json
```

Comment:
why are we doing this?

Reply:
I used the same approach as done just a few lines above for `coins`